src/lib/image/bootstrap/busybox/busybox.c

```diff
@@ -0,0 +1,45 @@
+/* 
```

Comment:
We should be able to consolidate these bootstrap modules into a single module that will look for a predefine path pattern (e.g. /usr/libexec/singularity/bootstrap/modules/%s.sh) so we don't need to create a new C module for every shell based counterpart.